xcm/src/v0/multi_location.rs

```diff
@@ -367,6 +367,31 @@ impl MultiLocation {
 		return self.at(prefix.len())
 	}
 
+	/// Returns whether `self` begins with or is equal to `prefix`.
+	///
+	/// # Example
+	/// ```rust
+	/// # use xcm::v0::{Junction::*, MultiLocation::*};
+	/// # fn main() {
+	/// let m = X4(Parent, PalletInstance(3), OnlyChild, OnlyChild);
+	/// assert!(m.matches_prefix(&X2(Parent, PalletInstance(3))));
+	/// assert!(m.matches_prefix(&m));
+	/// assert!(!m.matches_prefix(&X2(Parent, GeneralIndex(99))));
+	/// assert!(!m.matches_prefix(&X1(PalletInstance(3))));
+	/// # }
+	/// ```
+	pub fn matches_prefix(&self, prefix: &MultiLocation) -> bool {
```

Comment:
```suggestion
	pub fn starts_with(&self, prefix: &MultiLocation) -> bool {
```

Sounds more like the other methods available in rust